src/foam/core/Enum.js

```diff
@@ -181,6 +181,11 @@ foam.CLASS({
     function installInClass(cls) {
       var e = cls.create(this.definition);
       cls.installConstant(this.name, e);
+      if ( ! cls.getAxiomByName('FANCY') ) { cls.installAxiom(foam.core.Constant.create({ name: 'FANCY', value: false })); }
```

Comment:
Couldn't this just be added to the view, since it's the only one who cares?

Reply:
And then, just add it as a method isFancy(enum) that returns true if either color is set.

Reply:
We would wanted it to be assigned to an enum since the whole Enum should render the same way and this way we don't need to recalculate it every time we need to render an Enum value. 

If it was in a view we would need to iterate through all enum values each time.

Reply:
Assign it to the Enum, not the enumvalue and use the foam.stdlib.memoize1() method to have it cache the value. Just write the method isFancy(Enum) and I'll add the caching.